data/json/items/ammo/5x50.json

```diff
@@ -0,0 +1,40 @@
+[
+  {   
+    "id": "5x50dart",
+    "type": "AMMO",
+    "name": "RA110 5x50mm flechette",
+    "description": "Designed to defeat modern body armor, the Rivtech 5x50mm flechette round features a biodegradable sabot and a single, fin-stabilized penetrator.",
+    "weight": 2,
+    "volume": 1,
+    "price": 5000,
+    "count": 100,
+    "stack_size": 200,
+    "material": ["plastic", "powder"],
+    "ammo_type": "5x50",
+    "symbol": "=",
+    "color": "green",
+    "casing": "5x50_hull",
+    "damage": 20,
+    "pierce": 30,
+    "range": 60,
+    "dispersion": 100,
+    "recoil": 60,
+    "effects": ["COOKOFF", "NEVER_MISFIRES"]
```

Comment:
minor style issue (not sufficient to delay PR) should be space before array start/end